.github/workflows/rfs_pr_e2e_test.yml

```diff
@@ -0,0 +1,40 @@
+name: Jenkins E2E
+
+on:
+  push:
+    branches-ignore:
+      - 'backport/**'
+      - 'dependabot/**'
+  pull_request_target:
+    types: [opened, synchronize, reopened]
+
+env:
+  python-version: '3.11'
+
+permissions:
+  contents: read # to fetch code (actions/checkout)
+
+jobs:
+  rfs-e2e-aws-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Determine Github repository and branch
+        id: determine-repo-vars
+        run: |
+          if [[ "${GITHUB_EVENT_NAME}" == "pull_request_target" ]]; then
+            branch_name="${{ github.event.pull_request.head.ref }}"
+            pr_repo_url="https://github.com/${{ github.event.pull_request.head.repo.full_name }}.git"
+          else
+            branch_name="${{ github.ref_name }}"
+            pr_repo_url="https://github.com/${{ github.repository }}.git"
+          fi
+          echo "Running jenkins test on repo: $pr_repo_url and branch: $branch_name"
+          echo "branch_name=$branch_name" >> $GITHUB_OUTPUT
+          echo "pr_repo_url=$pr_repo_url" >> $GITHUB_OUTPUT
+      - name: Jenkins Job Trigger and Monitor
+        uses: lewijacn/jenkins-trigger@1.0.4
```

Comment:
Tip for next time; while iterating you can point to `main` during development until you ship a version that you are sure works, then create a release/tag

`uses: lewijacn/jenkins-trigger@main`